moto/ec2/models.py

```diff
@@ -341,12 +343,19 @@ def __init__(self):
         super(NetworkInterfaceBackend, self).__init__()
 
     def create_network_interface(
-        self, subnet, private_ip_address, group_ids=None, description=None, **kwargs
+        self,
+        subnet,
+        private_ip_address,
+        private_ip_addresses=None,
```

Comment:
I'm wondering if instead of adding a `private_ip_addresses`, you simply make `private_ip_address` a `list` that will take at least 1 value.

Reply:
You're right in the sense there doesn't seem to be a functional difference, but the (describe-network-interfaces)[https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.describe_network_interfaces]-method (not yet implemented) actually makes a distinction

Reply:
Oh interesting.